README.md

```diff
@@ -83,7 +83,6 @@ youtube2zim --api-key "<your-api-key>" --type user --id "Vsauce" --name "vsauce"
 
 ## Notes
 
-* Your API_KEY is subject to usage quotas (10,000 requests/day) so use `--only_test_branding` when adjusting parameters and branding to not *waste your quota*.
 * If you encounter issues reading ZIM files created using this scraper, please take a look at the [Compatibility Matrix](https://github.com/openzim/youtube/wiki/Compatibility) before opening a ticket.
```

Comment:
We should remove this statement, compatibility matrix, is now OK for all readers (compatibility matrix should be updated btw)

Reply:
Removed in [7f7c5bc](https://github.com/openzim/youtube/pull/271/commits/7f7c5bc44c001b2ec089372907973526c45f7ee4). 
Created #273 to track updating the compatibility matrix.